### ISSUE TITLE
[pwm] Have we supported PWM APIs on K64f？

### ISSUE THREAD

**Issue by cuiyanx** [01org/zephyr.js#1667]
### Description

The document [/docs/pwm.md](https://github.com/01org/zephyr.js/blob/master/docs/pwm.md) has be updated. But when I test PWM APIs on K64f, throw out an error. Have we supported it?

 The document description:
`FRDM-K64F For the K64F, you can use numbers 0-11 or strings "D3", "D5" through "D13", "A4" and "A5".`

Error logs:
`NotSupportedError: module not found: 'pwm'`

### Test Code

[/docs/pwm.md](https://github.com/01org/zephyr.js/blob/master/docs/pwm.md)

 

### Actual Result

Throw out error.
 

### Expected Result

Support on K64f.
 

### Test Builds

Branch | Commit Id | Target Device | Test Date | Result
---|---|---|---|---
master | dcf56a9 | FRDM-K64F | Nov 20, 2017 | Fail

**Comment by grgustaf** [comment]
No, the last I looked the PWM driver in Zephyr had not been updated to support the newer PWM apis, so it won't work. I have seen several references from QA as if it was working though, so I meant to check this out. So sounds like maybe not after all?
